src/components/SideBar.jsx

```diff
@@ -43,22 +37,18 @@ export default function ({ collections }) {
         }}
       >
         <Toolbar />
-        <Box sx={{ overflow: "auto", paddingLeft: 2 }}>
+        <Box sx={{ overflow: "auto", paddingLeft: 2, paddingTop: 5 }}>
```

Comment:
The COLLECTIONS header could use more padding at the top - it seems too close to the navbar now.